app/application/controller.js

```
@@ -23,5 +23,11 @@ export default Ember.Controller.extend({
     handleColorPropery(color) {
       console.log("color property : ", color);
     },
+    handleSelectedSlide(param) {
+      console.log("handle selected slide: ", param);
+    },
+    handleRemoveSlide() {
+      console.log('APEL CONTROLLER');
+    },
```

but if the controller doesn't actually save the removal, when you change to another slide you'll have the previously deleted slide back in the list. I would let the action bubble to the controller, save the delete and maybe have a redirectToRoute because the current route won t be valid anymore. I could be wrong tho'